src/components/shared/InfoPopover/MemberInfoPopover/MemberInfoPopover.tsx

```diff
@@ -4,26 +4,28 @@ import { BigNumber } from 'ethers';
 
 import { SpinnerLoader } from '~shared/Preloaders';
 import Tag from '~shared/Tag';
-import {
-  AnyUser,
-  useColonyNativeTokenQuery,
-  Colony,
-  useUserBalanceWithLockQuery,
-  useUserReputationForTopDomainsQuery,
-} from '~data/index';
+// import {
+//   AnyUser,
+//   useColonyNativeTokenQuery,
+//   Colony,
+//   useUserBalanceWithLockQuery,
+//   useUserReputationForTopDomainsQuery,
+// } from '~data/index';
```

Comment:
This can be removed entirely, as the `~data` import / functionality is not coming back